docs/section19/19-2.md

```diff
@@ -0,0 +1,74 @@
+            - Add CHW fluid loop to chilled water loop list: `CHW_fluid_loop_list =  CHW_fluid_loop_list.append(hvac.cooling_system.chilled_water_loop)`    
+            - Add condenser water fluid loop to condenser water loop list: `CW_fluid_loop_list =  CW_fluid_loop_list.append(hvac.cooling_system.condenser_water_loop)`  
+            - Add the child loops to the CHW loop list: `For CHW_child_loop in hvac.cooling_system.chilled_water_Loop.child_loops:`  
+                - Add to the list of chilled water loops: `CHW_fluid_loop_list =  CHW_fluid_loop_list.append(CHW_child_loop)`             
+- Cycle through each terminal unit in the B-RMR and create list of HW loops: `for terminal in B_RMR...Terminal:`  
+    - Reset list with the one terminal unit: `terminal_list = []` 
+    - Check if heating source is hot water: `if terminal.heating_source == "HOT_WATER":`  
+        - Create a list with the one terminal unit: `terminal_list = [terminal.id]`  
+        - Check if the loop is attached to a boiler: `if are_all_terminal_heating_loops_attached_to_boiler(B_RMR,terminal_list) ==  TRUE:`  
+            - Check if the fluid loop is already in the list of HW loops, if not then add: `if terminal.heating_from_loop Not in HW_fluid_loop_list: HW_fluid_loop_list.append(terminal.heating_from_loop)`  
+
+- Eliminate duplicates from all loop lists: `HW_fluid_loop_list = List(Set(HW_fluid_loop_list))`  
+- Eliminate duplicates from all loop lists: `CHW_fluid_loop_list = List(Set(CHW_fluid_loop_list))`  
+- Eliminate duplicates from all loop lists: `CW_fluid_loop_list = List(Set(CW_fluid_loop_list))`  
+
+**Applicability Check 1:**   
+- For each fluid_loop in B_RMR.ASHRAE229: `For fluid_loop in B_RMR.ASHRAE229:`  
+    - Check if the loop is in any of the lists created above, if it is carry on and if not then this rule is Not_Applicable for the fluid loop: `if fluid_loop in HW_fluid_loop_list OR fluid_loop in CHW_fluid_loop_list OR fluid_loop in CW_fluid_loop_list:`  
+        - Reset FluidLoopDesignAndControl_cool_b: `FluidLoopDesignAndControl_cool_b = ""`
+        - Reset FluidLoopDesignAndControl_heat_b: `FluidLoopDesignAndControl_heat_b = ""`  
+        - if the loop is in the CHW fluid loop: `if fluid_loop in CHW_fluid_loop_list:`  
+            - Get cooling_or_condensing_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_cool_b = fluid_loop.cooling_or_condensing_design_and_control`  
+        - Elif the loop is in the CW fluid loop: `if fluid_loop in CW_fluid_loop_list:`  
+            - Get cooling_or_condensing_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_cool_b = fluid_loop.cooling_or_condensing_design_and_control`  
+        - Elif the loop is in the HW fluid loop: `if fluid_loop in HW_fluid_loop_list:`  
+            - Get heating_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_heat_b = fluid_loop.heating_design_and_control`  
+
```

Comment:
I dont quite understand the applicability check here.
Does the actual check ends at this line?
`if fluid_loop in HW_fluid_loop_list OR fluid_loop in CHW_fluid_loop_list OR fluid_loop in CW_fluid_loop_list:`

Reply:
Yes exactly.

Reply:
Got it. Thanks